api/v1alpha1/addon_types.go

```diff
@@ -4,11 +4,24 @@
 package v1alpha1
 
 import (
+	corev1 "k8s.io/api/core/v1"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 
 	"github.com/d2iq-labs/cluster-api-runtime-extensions-nutanix/api/variables"
 )
 
+const (
+	AddonStrategyClusterResourceSet AddonStrategy = "ClusterResourceSet"
+	AddonStrategyHelmAddon          AddonStrategy = "HelmAddon"
+
+	VolumeBindingImmediate            string = "Immmediate"
+	VolumeBindingWaitForFirstConsumer string = "WaitForFirstConsumer"
+
+	VolumeReclaimRecycle string = "Recycle"
+	VolumeReclaimDelete  string = "Delete"
+	VolumeReclaimRetain  string = "Retain"
```

Comment:
nit: No need for string type here, it's implicit
```suggestion
	VolumeBindingImmediate            = "Immmediate"
	VolumeBindingWaitForFirstConsumer = "WaitForFirstConsumer"

	VolumeReclaimRecycle = "Recycle"
	VolumeReclaimDelete  = "Delete"
	VolumeReclaimRetain  = "Retain"
```

Reply:
ah nice!